### PR TITLE
Remove some hateful PEP8 errors

### PR DESCRIPTION
**Pull request by koriaf**
* and unused variables
* and spaces/indentation problems

### CODE REVIEW

File: zappa/zappa.py [hunk 1]
```diff
@@ -792,13 +784,13 @@ def create_and_setup_methods(self, api_id, resource_id, lambda_arn, report_progr
...
                 response_models = {content_type: 'Empty' for content_type in self.method_content_types}
 
-                method_response = self.apigateway_client.put_method_response(
-                        restApiId=api_id,
-                        resourceId=resource_id,
-                        httpMethod=method,
-                        statusCode=status_code,
-                        responseParameters=response_parameters,
-                        responseModels=response_models
+                self.apigateway_client.put_method_response(  # method_response = unused
```
Comment (Miserlou) [inline zappa/zappa.py]: No need for comment here

File: zappa/zappa.py [hunk 2]
```diff
@@ -822,31 +814,29 @@ def create_and_setup_methods(self, api_id, resource_id, lambda_arn, report_progr
...
 
-                integration_response = self.apigateway_client.put_integration_response(
-                        restApiId=api_id,
-                        resourceId=resource_id,
-                        httpMethod=method,
-                        statusCode=status_code,
-                        selectionPattern=self.selection_pattern(status_code),
-                        responseParameters=response_parameters,
-                        responseTemplates=response_templates
+                self.apigateway_client.put_integration_response(  # integration_response = unused
```
Comment (Miserlou) [inline zappa/zappa.py]: No need for comment here

File: zappa/zappa.py [hunk 3]
```diff
@@ -435,14 +438,13 @@ def splitpath(path):
     ##
 
     def upload_to_s3(self, source_path, bucket_name):
-        """
+        r"""
```
Comment (Miserlou) [inline zappa/zappa.py]: Why r"""?

File: zappa/zappa.py [hunk 4]
```diff
@@ -822,31 +814,29 @@ def create_and_setup_methods(self, api_id, resource_id, lambda_arn, report_progr
...
 
-        Returns the deployed API URL.
-
+        Return the deployed API URL.
         """
-
         print("Deploying API Gateway..")
 
-        response = self.apigateway_client.create_deployment(
+        self.apigateway_client.create_deployment(  # response =
```
Comment (Miserlou) [inline zappa/zappa.py]: Why

File: zappa/zappa.py [hunk 5]
```diff
@@ -907,7 +897,7 @@ def remove_api_key(self, api_id, stage_name):
             nameQuery='{}_{}'.format(stage_name, api_id)
         )
         for api_key in response.get('items'):
-            resp = self.apigateway_client.delete_api_key(
+            self.apigateway_client.delete_api_key(  # resp =
```
Comment (Miserlou) [inline zappa/zappa.py]: Why

File: zappa/zappa.py (hunk 3 above)
Comment (koriaf) [inline zappa/zappa.py]: For consistency, always use """triple double quotes""" around docstrings. Use r"""raw triple double quotes""" if you use any backslashes in your docstrings. For Unicode docstrings, use u"""Unicode triple-quoted strings""" . 

https://www.python.org/dev/peps/pep-0257/

File: zappa/zappa.py (hunk 1 above)
Comment (koriaf) [inline zappa/zappa.py]: yes, I just moved unused variables to comments with = character, so you can see this variable was unused and removed. To ensure it just was forgotten/wrongly assigned and no hidden bugs here.

**Comment by koriaf** [comment]
Shall I remove all these comments? All why-s covered by my first answer?

**Comment by Miserlou** [comment]
Yeah, I'd prefer it if you could take the comments out please. The r""" is okay, I just hadn't seen that before and thought it might be a typo. Sorry about the need to rebase but I wanted to merge in the other PR for today's release.

**Comment by koriaf** [comment]
Okay. I can't rebase without force-pushing the repo and I can't merge because there will be some merge commits. So please destroy this PR, I'll create the new one soon.

**Comment by Miserlou** [comment]
Roger that

**Pull request closed without merging**